src/App.spec.ts

```diff
@@ -260,16 +277,15 @@ describe('App', () => {
       // Act
       const app = new App({ receiver: fakeReceiver, logger: fakeLogger, authorize: noopAuthorize });
       app.use(fakeMiddleware);
-      for (const event of invalidReceiverEvents) {
-        fakeReceiver.emit('message', event);
-      }
 
+      await Promise.all(invalidReceiverEvents.map(event => fakeReceiver.sendEvent(event)));
       // Assert
       assert(fakeErrorHandler.notCalled);
       assert(fakeMiddleware.notCalled);
       assert.isAtLeast(fakeLogger.warn.callCount, invalidReceiverEvents.length);
     });
-    it('should warn, send to global error handler, and skip when a receiver event fails authorization', async () => {
+
+    it('should warn and skip when a receiver event fails authorization', async () => {
```

Comment:
as per comment in `App.ts`, is there a reason we are removing the global error handler from the chain of error handling?

Reply:
I can't find the referenced comment, the global error handler for middleware is here: https://github.com/slackapi/bolt/pull/380/files#diff-0286657a0789ea9446fa3de979ff3e09R571

I don't believe I deleted a test that tested this. Could you link to that if you have one in mind?

I'm converting the test case that checked for `ack` being called with an error to explicity check that middleware errors go to the global handler.

Reply:
It will be called `should, on error call the global error handler` for your cmd+f

Reply:
the new test case seems useful, thanks!

however, this test case has to do specifically with authorization failure. and this PR is essentially removing a bunch of assertions that verify the global error handler is called in the cause of an authorization failure. i think that's behavior that we want to preserve. if you have a reason in mind for altering that behavior, i'd like to understand that.